File: exs/mundo_1/golang/008.go
/*
	Desafio: 008
	Problema: Escreva um programa que leia um valor 
	em metros e o exiba convertido em centímetros 
	e milímetros.
*/

package main
import "fmt"

func main() {
	var meter float64
	var centimeter float64
	var millimeter float64
	fmt.Print("Informe o valor em metro(s): ")
	fmt.Scanln(&meter)
	centimeter = meter * 100
	millimeter = meter * 1000
	fmt.Println("Centímetro:", centimeter)
	fmt.Println("Milímetro:", millimeter)
}

File: exs/mundo_1/rust/008.rs
/*
Desafio 008

Problema: Escreva um programa que leia um valor em metros e o exiba convertido em
          centímetros e milímetros.
*/

fn main() {
    let mut metro = String::new();

    println!("Digite um valor em metros: ");
    std::io::stdin().read_line(&mut metro).unwrap();

    let m: f32 = metro.trim().parse().expect("Erro ao ler valor");

    let dc = metro * 10.0;
    let cm = metro * 100.0;
    let mm = metro * 1000.0;

    println!("{m} metros é igual a {dm} decametros, {cm} centímetros e {mm} milímetros",);
}
